packs/github/actions/deployment_event.yaml

```diff
@@ -0,0 +1,50 @@
+---
+name: "deployment_event"
+description: "Process an deployment event from GitHub."
```

Comment:
Maybe a more description name and description is not such a bad idea.

From just looking at the name / description I don't really have an idea what action is supposed to do. Description from the workflow ("A workflow to process an github deployment event and install a pack if the environment matches.") should be sufficient.